package.json

```diff
@@ -98,6 +98,7 @@
     "i18n:compile": "lingui compile",
     "i18n": "yarn i18n:extract && yarn i18n:compile",
     "start": "react-app-rewired start",
+    "start-local": "env-cmd -f .env.local react-app-rewired start",
```

Comment:
we dont have `.env.local` file, do we? @nguyenhuudungz

Reply:
I have .env.local file but don't push it. I like 50/50 when pushing this code.

Reply:
Dont get your idea

Reply:
What do you mean of `50/50 when pushing this code`

Reply:
When I dev backend, I have a lot of services in localhost. So creating .env.local is a convenient way for me to work around it. I think it doesn't bored you right.

Reply:
okay bro, understand